bin/package.json

```diff
@@ -0,0 +1 @@
+{ "type": "commonjs" }
```

Comment:
Any reason not just use ESM here? I could have sworn ESM worked in these extensionless files...

Reply:
It's been awhile since I wrote this bit, but it wasn't working at the time.